draft-ietf-taps-interface.md

```diff
@@ -3419,8 +3419,25 @@ The Transport Services API  provides the following guarantees about the ordering
 
 RFC-EDITOR: Please remove this section before publication.
 
-This document has no actions for IANA.
-Later versions of this document may create IANA registries for generic transport property names and transport property namespaces (see {{property-names}}).
+IANA will create the following registry in a new registry group called "Transport Service System (TAPS)":
+
+Registry Name: Transport Property Namespaces
+
+Registration Procedure:
+
+- IETF Review
+- First Come First Served for vendor/implementation specific entries strating with an underscore "_"
+
+The initial values in the "Transport Property Namespaces" registry are:
+
+   | namespace  | Protocol                              | Reference       |
+   |------------|---------------------------------------|-----------------|
+   | udp        | User Datagram Protocol                | This Document   |
+   | tcp        | Transmission Control Protocol         | This Document   |
+   | sctp       | Stream Transmission Control Protocol  | This Document   |
```

Comment:
Ought we to add UDP-Lite?

Reply:
That is a good question … we actually only specify protocol specific properties for TCP and do not explicitly state that there are none for UDP and SCTP. 
So either remove UDP and SCTP form the list or add UDP-Lite and add sentence that there are no protocol specific properties because all tuneable they are mapped from generic properties.

Reply:
I'd say: Either all the protocols we specify, or only the ones we touch :-)

Reply:
Fixed – only include the ones we touch.